BMI Calculator/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+  "manifest_version": 2,
+  "name": " BMI Calculator",
+  "version": "0.0.1",
+  "icons": {
+    "100": "./Logo-code.png"
+  },
+  "browser_action": {
+    "default_popup": "index.html"
```

Comment:
This extension is not loading properly, bcz you have used index.html in default_popup but the file is named as popup.html
So make it popup.html instead of index.html

Reply:
okk thankyou mam/sir